src/vg.proto

```diff
@@ -120,6 +120,7 @@ message Alignment {
     repeated Path fragment = 17; // An estimate of the length of the fragment, if this Alignment is paired.
     repeated Locus locus = 18; // The loci that this alignment supports. TODO: get rid of this, we have annotations in our data model again.
     repeated Position refpos = 19; // Position of the alignment in reference paths embedded in graph
+    double uniqueness = 20; // Meant to encode the uniqueness of the mapping based on its seeds
```

Comment:
We should explain what uniqueness is here. Is 0 "not unique" and 1 "completely unique"? What does a uniqueness of 0.5 mean?